### ISSUE TITLE
textarea - no announcement of maxlength

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/VFO-standards-support#300]
### Summary

JAWS doesn't stops announcing typed characters after max length of text area was reached. 

Example:
1. Open example in any browser
2.  Navigate into the textarea
3. type more characters then is maxlength 
4. keep typing even you will hit maxlenght

### Expected result
JAWS stops narrating characters as they are typed.  (at least it is what voiceOver does when max length is reached)

### Actual result
JAWS continues to narrate characters as they are typed. There is no way for the user to know that the characters are being discarded.

### Example
`<!DOCTYPE html>
<html>
<body>
<textarea rows="4" cols="50" maxlength="5">
</textarea>
</body>
</html>`

#### JAWS version and build number
2019.1907.42
 
#### Operating System and version
OS10

#### Browser and version:
Chrome 76

**Comment by JAWS-test** [comment]
The problem occurs not only with textarea, but also with <code>input type=text</code>.

**Comment by kurtjwolf** [comment]
An enhancement request has been made internally with bug #104711

**Comment by japower5** [comment]
This continues to be an issue in JAWS23.  Can we find out when it will be addressed?

**Comment by stevefaulkner** [comment]
Still an issue in JAWS 2024

**Comment by BrettLewisVispero** [comment]
We have tried to address this one a couple of times and not come up with a satisfactory implementation. Maybe not echoing chars would be better than nothing.

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 79622

**Comment by kaukeshav** [comment]
Hello Team, Is JAWS team working on this bug as it's been move to triaged status?
- If you can share in which release we are expecting this issue would addressed in JAWS. 
- Also, would be it fixed for other types of input like input[type='text'] ?

**Comment by BrettLewisVispero** [comment]
Hi,
We do have a fix for this in our internal builds. I think it will be in the July release of JAWS depending on testing.
We are generically looking for the maxlength attribute so any field that supports this should work.

**Comment by Kseleny** [comment]
Hi @BrettLewisVispero - Can you please provide an update on this item and whether it will be remediated in the July release?  Lots of eyes on this one!  TY!

**Comment by BrettLewisVispero** [comment]
Yes, this should be available in the July update for Chrome. We are not able to access the maxlength attribute in Firefox.

**Comment by Kseleny** [comment]
That is fantastic news!  Can you please confirm the release date for me?  We plan to do validation testing here at Salesforce ASAP after the feature is live.  TY!!

**Comment by jonathan-cavey** [comment]
@BrettLewisVispero we are also keeping a close eye on this, any update on that release date? 
Is this update based on a Chrome release? OR a JAWS Release?

**Comment by BrettLewisVispero** [comment]
The just released July JAWS update should have the maxlength support in Chrome. Please let me know if it isn't working for some reason.

**Comment by satyaminnekanti** [comment]
tested on this [page](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/maxlength) with sample for **maxlength** and with chrome and JAWS not providing about any update of maxlength reached or characters not typing in the field
